provider/aws/lambda/autoscale/handler.go

```diff
@@ -298,7 +313,12 @@ func desiredCapacity(largest, total *Metrics) (int64, error) {
 	debug("desired = %+v\n", desired)
 
 	// minimum instance count is 3
-	if desired < 3 {
+	if IsHA == "true" && desired < 3 {
+		desired = 3
+	}
+
+	// for no-HA rack the max is 3
+	if IsHA == "false" && desired > 3 {
 		desired = 3
 	}
```

Comment:
1 - I think it's fair to keep it at 3 for non HA - the whole point of non HA clusters is to prevent extra costs so if anyone is looking for more computing power, they should go with HA racks.

2 - Still reviewing it